server/cmwell-ws/app/controllers/InputHandler.scala

```diff
@@ -62,10 +62,14 @@ class InputHandler @Inject() (ingestPushback: IngestPushback,
    */
   def handlePost(format: String = "") = ingestPushback.async(parse.raw) { implicit req =>
     RequestMonitor.add("in", req.path, req.rawQueryString, req.body.asBytes().fold("")(_.utf8String))
-    val resp = if ("jsonw" == format.toLowerCase) handlePostWrapped(req) -> Future.successful(Seq.empty[(String,String)]) else handlePostRDF(req)
-    resp._2.flatMap { headers =>
-      keepAliveByDrippingNewlines(resp._1,headers)
-    }.recover(errorHandler)
+    if (req.getQueryString("priority").isDefined && !authUtils.isOperationAllowedForUser(security.PriorityWrite, authUtils.extractTokenFrom(req), evenForNonProdEnv = true)) {
```

Comment:
Also, and fix me if I'm wrong here, `authUtils.isOperationAllowedForUser` could be expensive, so order of checks matter. maybe we should document this...